octobot_trading/producers/ohlcv_updater.py

```diff
@@ -82,15 +85,26 @@ async def _initialize_candles(self, time_frame, pair):
         candles: list = await self.channel.exchange_manager.exchange \
             .get_symbol_prices(pair, time_frame, limit=self.OHLCV_OLD_LIMIT)
         self.channel.exchange_manager.uniformize_candles_if_necessary(candles)
-        if candles:
+        if len(candles) > 1:
```

Comment:
https://docs.python.org/3/library/stdtypes.html#truth-value-testing

Reply:
I don"t understand what you mean, here we want to check if the list has a size >= 2

Reply:
oh yeah my bad